ddl/ddl_api.go

```diff
@@ -461,6 +461,14 @@ func checkColumnDefaultValue(ctx sessionctx.Context, col *table.Column, value in
 	return hasDefaultValue, value, nil
 }
 
+func checkSequenceDefaultValue(col *table.Column) error {
+	if col.Tp == mysql.TypeTiny || col.Tp == mysql.TypeShort || col.Tp == mysql.TypeInt24 || col.Tp == mysql.TypeLong || col.Tp == mysql.TypeLonglong {
+		return nil
+	}
+	// TODO: refine the error.
+	return errors.New("column type is not suitable for default sequence value")
```

Comment:
Please refine this error, you can use a existing error code, for example `8200`.